tsconfig.json

```
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es5",
```

The [documentation of `ncc` says](https://github.com/vercel/ncc?tab=readme-ov-file#with-typescript):

> Most likely you want to indicate es2015 support:

es2015 is es6. Now, that's a good question, which one to use?

This action is guaranteed to run on Node 20. According to https://node.green/ and https://compat-table.github.io/compat-table/es6/ yes, Node 20 supports all the es6 features. So we can safely change the target to es6. Also let's follow the module resolution recommended by ncc:

```suggestion
    "module": "node16",
    "moduleResolution": "node16",
    "target": "es2015",
```